### PR TITLE
Animal table

### PR DESCRIPTION
**Pull request by anita00001**
# In this PR, we have:

## Basic Setup
- Used [curriculum-template-databases](https://github.com/microverseinc/curriculum-template-databases) to generate new repository
- Created a database named vet_clinic

## Database Table
- Created a table named `animals` with the following columns:
    - `id`: integer
    - `name`: string
    - `date_of_birth`: date
    - `escape_attempts`: integer
    - `neutered`: boolean
    - `weight_kg`: decimal
- Copied the SQL for creating a table into the schema.sql file

## Insert Data
image
- Copied the SQL for inserting data into the table in the data.sql file

## Write Queries
- Find all animals whose name ends with "mon"
image

- List the name of all animals born between 2016 and 2019
image

- List the name of all animals that are neutered and have less than 3 escape attempts
image

- List the date of birth of all animals named either "Agumon" or "Pikachu"
image

- List name and escape attempts of animals that weigh more than 10.5kg
image

- Find all animals that are neutered
image

- Find all animals not named Gabumon
image

- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with weights that equals precisely 10.4kg or 17.3kg)
image

- Copied the SQL for queries of the quests into the queries.sql file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,245 +8,122 @@ After you're finished please remove all the comments and instructions!
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (cynthiainga) [inline README.md]: - [x] Good job so far but the MIT License link is broken kindly fix that, having broken links in README file doesn’t look more professional 😎 🙃

File: schema.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY, 
+    name varchar,
```
Comment (cynthiainga) [inline schema.sql]: - [x] The varchar length should be specified and the maximum is `255`, please kindly fix that 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @anita00001, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Screenshots of queries result added ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇🏾 👇🏾 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @anita00001👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Good job with the animals' table. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**